config/settings/local.py

```diff
@@ -1,3 +1,7 @@
+import mimetypes
+
 from .base import *  # NOQA
 
 INSTALLED_APPS = INSTALLED_APPS + ["django_extensions"]  # NOQA
+
+mimetypes.add_type("text/css", ".css", True)
```

Comment:
@jerivas I don't know much about this, but I found it on SO and it seems to work for local dev 😂

Reply:
Never seen it before. What's it for?

Reply:
Hm, initially in local dev the server was serving the CSS file with `text/html` instead of `text/css` -- but now I can't reproduce it, so. 🤷